cmake/modules/AddSwift.cmake

```diff
@@ -1387,15 +1390,62 @@ endfunction()
 #   Sources to add into this library.
 function(add_swift_library name)
   set(SWIFTLIB_options
-      SHARED STATIC OBJECT_LIBRARY IS_STDLIB IS_STDLIB_CORE IS_SDK_OVERLAY
-      TARGET_LIBRARY FORCE_BUILD_FOR_HOST_SDK
-      API_NOTES_NON_OVERLAY DONT_EMBED_BITCODE HAS_SWIFT_CONTENT
-      FORCE_BUILD_OPTIMIZED NOSWIFTRT)
+        API_NOTES_NON_OVERLAY
+        DONT_EMBED_BITCODE
+        FORCE_BUILD_FOR_HOST_SDK
+        FORCE_BUILD_OPTIMIZED
+        HAS_SWIFT_CONTENT
+        IS_SDK_OVERLAY
+        IS_STDLIB
+        IS_STDLIB_CORE
+        NOSWIFTRT
+        OBJECT_LIBRARY
+        SHARED
+        STATIC
+        TARGET_LIBRARY)
+  set(SWIFTLIB_single_parameter_options
+        DEPLOYMENT_VERSION_IOS
+        DEPLOYMENT_VERSION_OSX
+        DEPLOYMENT_VERSION_TVOS
+        DEPLOYMENT_VERSION_WATCHOS
+        INSTALL_IN_COMPONENT)
+  set(SWIFTLIB_multiple_parameter_options
+        C_COMPILE_FLAGS
+        DEPENDS
+        FILE_DEPENDS
+        FRAMEWORK_DEPENDS
+        FRAMEWORK_DEPENDS_IOS_TVOS
+        FRAMEWORK_DEPENDS_OSX
+        FRAMEWORK_DEPENDS_WEAK
+        INCORPORATE_OBJECT_LIBRARIES
+        INCORPORATE_OBJECT_LIBRARIES_SHARED_ONLY
+        INTERFACE_LINK_LIBRARIES
+        LINK_FLAGS
+        LINK_LIBRARIES
+        LLVM_COMPONENT_DEPENDS
+        PRIVATE_LINK_LIBRARIES
+        SWIFT_COMPILE_FLAGS
+        SWIFT_COMPILE_FLAGS_IOS
+        SWIFT_COMPILE_FLAGS_OSX
+        SWIFT_COMPILE_FLAGS_TVOS
+        SWIFT_COMPILE_FLAGS_WATCHOS
+        SWIFT_MODULE_DEPENDS
+        SWIFT_MODULE_DEPENDS_CYGWIN
+        SWIFT_MODULE_DEPENDS_FREEBSD
+        SWIFT_MODULE_DEPENDS_HAIKU
+        SWIFT_MODULE_DEPENDS_DARWIN
+        SWIFT_MODULE_DEPENDS_IOS
+        SWIFT_MODULE_DEPENDS_LINUX
+        SWIFT_MODULE_DEPENDS_OSX
+        SWIFT_MODULE_DEPENDS_TVOS
+        SWIFT_MODULE_DEPENDS_WATCHOS
+        TARGET_SDKS)
+
   cmake_parse_arguments(SWIFTLIB
-    "${SWIFTLIB_options}"
-    "INSTALL_IN_COMPONENT;DEPLOYMENT_VERSION_OSX;DEPLOYMENT_VERSION_IOS;DEPLOYMENT_VERSION_TVOS;DEPLOYMENT_VERSION_WATCHOS"
-    "DEPENDS;LINK_LIBRARIES;SWIFT_MODULE_DEPENDS;SWIFT_MODULE_DEPENDS_OSX;SWIFT_MODULE_DEPENDS_IOS;SWIFT_MODULE_DEPENDS_TVOS;SWIFT_MODULE_DEPENDS_WATCHOS;SWIFT_MODULE_DEPENDS_FREEBSD;SWIFT_MODULE_DEPENDS_LINUX;SWIFT_MODULE_DEPENDS_CYGWIN;SWIFT_MODULE_DEPENDS_HAIKU;FRAMEWORK_DEPENDS;FRAMEWORK_DEPENDS_WEAK;FRAMEWORK_DEPENDS_OSX;FRAMEWORK_DEPENDS_IOS_TVOS;LLVM_COMPONENT_DEPENDS;FILE_DEPENDS;TARGET_SDKS;C_COMPILE_FLAGS;SWIFT_COMPILE_FLAGS;SWIFT_COMPILE_FLAGS_OSX;SWIFT_COMPILE_FLAGS_IOS;SWIFT_COMPILE_FLAGS_TVOS;SWIFT_COMPILE_FLAGS_WATCHOS;LINK_FLAGS;PRIVATE_LINK_LIBRARIES;INTERFACE_LINK_LIBRARIES;INCORPORATE_OBJECT_LIBRARIES;INCORPORATE_OBJECT_LIBRARIES_SHARED_ONLY"
-    ${ARGN})
+                        "${SWIFTLIB_options}"
```

Comment:
Standard CMake styling does not require that arguments are "visually" indented (like in Python) to the same column as the first argument. In fact, the standard style in this project is all subsequent lines are to be indented 2 spaces. These sorts of whitespace changes and re-stylings add a lot of diff churn and potential merge conflicts.